server/controllers/locationController.ts

```diff
@@ -0,0 +1,95 @@
+import { Request, Response } from 'express';
+import { Like } from 'typeorm';
+
+import { Location } from 'server/models/Location';
+
+export default {
+  async index(req: Request, res: Response) {
+    const { country_code, city, region, postal_code, address } = req.query;
+
+    const locations = await Location.find({
+      where: [
+        { country_code: Like(`%${country_code}%`) },
+        { city: Like(`%${city}%`) },
+        { region: Like(`%${region}%`) },
+        { postal_code: Like(`%${postal_code}%`) },
+        { address: Like(`%${address}%`) },
+      ],
+    });
+
+    res.json(locations);
+  },
+
+  async show(req: Request, res: Response) {
+    const { id } = req.params;
+
+    const location = await Location.findOne({ id: parseInt(id) });
```

Comment:
Are we not requiring radix for `parseInt` calls?

Reply:
Do we really need it?

Reply:
Not really, I'm more curious why it wasn't picked up by the linter (it wasn't excluded in the rules and I remember it being a default; maybe we're using a different base)

Reply:
I don't know lol, but I don't think we should add it here, because it's a kind of verbose and unnecessary